config/bts.php

```diff
@@ -37,7 +37,7 @@
         'events' => [
             'accepted_external' => [
                 'from'  => ['pm@bts-crew.com'],
-                'to'    => ['P.Brooks@bath.ac.uk'],
+                'to'    => ['cw887@bath.ac.uk'],
```

Comment:
I presume we don't need sign-off to change this one as well?

Reply:
Good question - will seek and confirm

Reply:
Yep, we've got approval to make this change as well from Oli Tailby :)